### PR TITLE
Use cloudinary.v2.uploader.explicit instead of cloudinary.v2.api.resource

### PR DESCRIPTION
**Pull request by eexit**
Hello Marco,

I'm recently trying to migrate all my images to Cloudinary so I'm diving into your code to see how it works. First of all, thank you for your work here.

Here a small PR to improve the behavior of `exists`:

- Returns explicit a boolean (to follow Ghost storage interface)
- Use `cloudinary.v2.uploader.explicit` because it's not rate limited as opposed as `cloudinary.v2.api.resource`

> Note that if you only need details about the original resource, you can also use the upload or explicit methods, which are not rate limited.

PS: you find a better way to review the PR with the [?w=1 param](https://github.com/mmornati/ghost-cloudinary-store/pull/12/files?w=1).

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -14,9 +14,10 @@ class CloudinaryAdapter extends BaseAdapter{
 
   exists(filename) {
     return new BlueBird(function(resolve) {
-      cloudinary.v2.api.resource(path.parse(filename).name, {type: 'upload'}, function(error, result) {
+      // Use explicit instead of resource because there's no rate limit for explicit
+      cloudinary.v2.uploader.explicit(path.parse(filename).name, {type: 'upload'}, function(error, result) {
         if (result) {
-          resolve(result);
+          resolve(true);
```
Comment (mmornati) [inline index.js:20]: We don't need the result here (for ghost)? With only the true  resolve are we able to link the image to the post?

**Review by mmornati — COMMENTED** [review]
Good job and thanks. I'll give it a try this evening.

File: index.js (hunk 1 above)
Comment (eexit) [reply index.js]: The image URL in the post is returned by the `save` method. The `exists` method only returns a boolean.
However, I could just return the boolean without the resolve.

**Comment by eexit** [comment]
@mmornati I just pushed an almost complete re-write of the adapter. I added some more tests and added support for image upload into folder.
https://github.com/eexit/ghost-storage-cloudinary

**Review by mmornati — APPROVED** [review]

**Comment by mmornati** [comment]
@eexit good job in all your refactor. Can I integrate all of this in mine one too?

**Pull request merged**